content/300-guides/500-other/900-advanced-database-tasks/06-typesafe-raw-sql/01-safeql-gis.mdx

```diff
@@ -0,0 +1,183 @@
+---
+title: 'Using SafeQL with unsupported GIS features'
+metaTitle: 'Using SafeQL with unsupported GIS features (Guide)'
+metaDescription: 'Learn how to use SafeQL and Prisma Client extensions to work around features not natively supported by Prisma.'
```

Comment:
We work in PostGIS or geographical data to improve SEO in the title and description 😄